Dacaspex/Exhibit
# Feature - Clearing of data representation and scene

#### What have I done
Implemented the clear button to actually clear the data representation and scene.
- Added `ClearService` that has dependencies to all components that need to be cleared
- Added `clear()` or `reset()` functions to required components.

#### Testing
- Import files, run the simulation for a while, select some sheets and POIs. Then clear and observe that everything works as expected.
- Import files with multiple timing files. Do the same as above.
- Change the camera / scene settings, do the same as above. 

#### Addresses SRD section(s)
- `ConfigurationService`
- `PrinterModel`
- `CameraProviderService`
- Added `ClearService`
- `TimeService`
